examples/bq-remote-function/README.md

```diff
@@ -0,0 +1,18 @@
+# BigQuery Remote Function Java Samples
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/K4singh/professional-services&page=editor&open_in_editor=examples/bq-remote-function/string_format
+
+[Bigquery remote functions](https://cloud.google.com/bigquery/docs/reference/standard-sql/remote-functions) allows you to use your custom services or libraries written in any language other than SQL and javascript which are not present as bigquery user defined functions. 
+This repository has sample code for setting up bigquery remote function. it contains simple string format code, the custom function is deployed as cloud run gen 2 which can be invoked using api call.
```

Comment:
What does the string formatting do in this case? Please provide a simple example of the transformation

Reply:
add example in deployment steps